packages/web-app-files/src/components/SideBar/Shares/Collaborators/CollaboratorsEditOptions.vue

```diff
@@ -274,6 +314,9 @@ export default {
     } else {
       this.selectedRole = this.roles[0]
     }
+
+    this.getRecipientsCustomPermissions()
+    window.addEventListener('keydown', this.cycleRoles)
```

Comment:
The event listener also get's added in the `mounted` hook. Which one do you want to keep? Please remove the other one.